views/segments.html

```diff
@@ -173,6 +173,14 @@
     'wooNumberOfOrdersInTheLast': _x('in the last', 'Appears together with wooNumberOfOrdersDays when creating a new WooCommerce segment based on the number of orders.'),
     'wooPurchasedCategory': __('purchased in this category'),
     'wooPurchasedProduct': __('purchased this product'),
+    'wooTotalSpent': __('total spent'),
+    'wooTotalSpentMoreThan': __('more than'),
+    'wooTotalSpentLessThan': __('less than'),
+    'wooTotalSpentAmount': __('amount'),
+    'wooTotalSpentDaysPlaceholder': __('days'),
+    'wooTotalSpentDays': _x('days', 'Appears together with wooTotalSpentInTheLast when creating a new WooCommerce segment based on the total spent money.'),
+    'wooTotalSpentCurrency': __('$'),
```

Comment:
I am not sure what can translators do about this. Do we need to translate this? And if so how? At least there should be a comment, but I am not sure about it.

Reply:
This is the rest of the first code. I will remove it. Thanks